### ISSUE TITLE
Popup Menu Overflows

### ISSUE THREAD

**Issue by manno** [manno/gnome-shell-extension-quicklaunch#1]
If the user has to many applications in the list the menu will simply overflow. Solution: add scrollbars to the menu if menu size > screen size?

**Comment by lachlan-00** [comment]
yes, list needs to be scrollable. i have too many items for rdp connections to local servers. It took me a while to realise but it was popping open off screen after a limit was hit.
